plugins/modules/cloud/scaleway/scaleway_compute.py

```diff
@@ -664,6 +678,12 @@ def main():
     module = AnsibleModule(
         argument_spec=argument_spec,
         supports_check_mode=True,
+        mutually_exclusive=[
```

Comment:
They are not mutually exclusive, you can define both and the API will accept it.

Reply:
https://developers.scaleway.com/en/products/instance/api/#post-7482b1 claims "Only one of `organization` and `project` may be set.".

Reply:
If I set project and organization I have this error:
 ![image](https://user-images.githubusercontent.com/96626847/147553473-26e2c44e-0cb4-408b-a3e5-9b164bae6d52.png)

and according to https://developers.scaleway.com/en/products/instance/api/#post-7482b1 "Only one of organization and project may be set".

Can you check if the api in your side really accept both values  ?

Reply:
Ah yes sorry, it seems that the period where both were accepted is over. My bad.